Storage/StorageTests/Tools/FileStorageTests.swift

```diff
@@ -0,0 +1,29 @@
+import XCTest
+@testable import Storage
+
+final class FileStorageTests: XCTestCase {
+    private var fileURL: URL?
+    private var subject: PListFileStorage?
+
+    override func setUp() {
+        super.setUp()
+        subject = PListFileStorage()
+        fileURL = Bundle(for: FileStorageTests.self)
+            .url(forResource: "shipment-provider", withExtension: "plist")
+    }
+
+    override func tearDown() {
+        fileURL = nil
+        subject = nil
```

Comment:
Does `super.tearDown()` belong here? I can't remember if it's required on tearDown().

Reply:
🤦‍♂️  thanks!